airbyte-integrations/connectors/destination-bigquery/build.gradle

```diff
@@ -12,9 +12,8 @@ application {
 dependencies {
```

Comment:
I forgot to mention this on the previous S3/GCS PR but can we make the integration tests of warehouses destinations (bigquery / snowflake / etc in the future) dependent from blob storage destinations that are used underneath for staging?

So if we run `./gradlew :airbyte-integrations:connectors:destination-gcs:integrationTest`, it should also run: `./gradlew :airbyte-integrations:connectors:destination-bigquery*:integrationTest`, etc?

Reply:
Good idea. I created an issue here: https://github.com/airbytehq/airbyte/issues/11815